AV Status/AVStatus.vbs

```diff
@@ -2377,7 +2383,7 @@ Sub ObtainKESServerData
         If CInt(Left(FormattedAVVersion,InStr(FormattedAVVersion,".")-1)) <= 10 Then
             FormattedPatternAge = DateValue(Mid(RawAVDate,7,4) & "/" & Mid(RawAVDate,4,2) & "/" & Left(RawAVDate,2) )
         Else
-            If GetLocale() = 2057 Then  'If the device is in the UK, let's transpose the day and month, so that we get things right.  
+            If (GetLocale() = 2057 OR GetLocale()=1031 OR GetLocale()=1043) Then  'If the device is in the UK, Germany or the Netherlands, let's transpose the day and month, so that we get things right.  

```

Comment:
Also Belgium writes the dates in dd/mm/yyyy
If I do getlocale() in a vbs script on my machine , I get 2067 as code.

Reply:
You may want to tell Chris Reid about that, he wrote the script. I just figured I'd get the latest version on here.

Reply:
Have updated the script to 2.36, and updated the comment to include Belgium